docs/dev-guide/python/timers.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: timers
+title: Timers
+sidebar_label: Timers
+sidebar_position: 17
```

Comment:
Don't really need the sidebar_position metadata anymore because of the explicit sidebar.js config.